### PR TITLE
initial pipeline to build bosh release

### PR DESCRIPTION
**Pull request by m-richo**

### CODE REVIEW

File: ci/pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    initial_version: 0.3.0
+    private_key: {{git_boshreleases_private_key}}
+
+jobs:
+- name: build-it
+  plan:
+  - aggregate:
+    - get: version
+      params:
+        bump: patch
```
Comment (gordcorp) [inline ci/pipeline.yml]: Is it also possile to have a different build job for bumping the major/minor version numbers as well?

**Review by gordcorp — APPROVED** [review]
LGTM, minor question

File: ci/pipeline.yml (hunk 1 above)
Comment (m-richo) [reply ci/pipeline.yml]: good question.  i think it depends on how much we want to adopt semantic versioning.  many BOSH releases don't bother.

File: ci/create-bosh-release.sh [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  echo "v${VERSION}"                 > ${RELEASE_ROOT}/tag
+  echo "${RELEASE_NAME} v${VERSION}" > ${RELEASE_ROOT}/name
+  cat > ${RELEASE_ROOT}/body         << EOF
+## Manifest
+\`\`\`
+releases:
+- name: $RELEASE_NAME
+  version: $VERSION
+  sha1: $RELEASE_SHA1
+  url: https://github.com/govau/${RELEASE_NAME}/releases/download/v${VERSION}/${RELEASE_NAME}-release-${VERSION}.tgz
```
Comment (m-richo) [inline ci/create-bosh-release.sh]: i think this release name is consistent with some of our other BOSH releases, but not with previous manually created releases of `dta-frontend`

**Pull request merged**